### ISSUE TITLE
Are compound autocomplete tokens conforming?

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag#1697]
In Input purposes we provide a list of tokens. But what if there are more than one of those on a page.
For instance an address for shipping and billing. For this reason HTML 5 has compound autocompletes that are space delimited.

https://html.spec.whatwg.org/#autofilling-form-controls:-the-autocomplete-attribute''

They provide this example:
For example, if there are two shipping addresses in the form, then they could be marked up as:
```
<fieldset>
 <legend>Ship the blue gift to...</legend>
 <p> <label> Address:     <textarea name=ba autocomplete="section-blue shipping street-address"></textarea> </label>
 <p> <label> City:        <input name=bc autocomplete="section-blue shipping address-level2"> </label>
 <p> <label> Postal Code: <input name=bp autocomplete="section-blue shipping postal-code"> </label>
</fieldset>
<fieldset>
 <legend>Ship the red gift to...</legend>
 <p> <label> Address:     <textarea name=ra autocomplete="section-red shipping street-address"></textarea> </label>
 <p> <label> City:        <input name=rc autocomplete="section-red shipping address-level2"> </label>
 <p> <label> Postal Code: <input name=rp autocomplete="section-red shipping postal-code"> </label>
</fieldset>
```
AXE/WAVE are not currently failing this. I think it should pass since its legal HTML and since to my knowledge no software has appeared on the market to do anything with these input purposes yet.

**Comment by JAWS-test** [comment]
My understanding of SC 1.3.5 is that the compount tokens are not required, but they are not a violation either

**Comment by mraccess77** [comment]
I agree they would not be required and would not fail if you did not use them.  Also I'd like to point out that I don't think automated tools should be failing autocomplete values that are not valid on form fields that are not collecting information about the user.  The SC is only scoped to input collecting information about the user - if a field not about the user has an autocomplete attribute that is wrong - it's not a fail of the criterion but cause the criterion doesn't apply.

**Comment by fstrr** [comment]
This issue looks like it can be closed. If it needs to be re-opened, please do that and convert it to a Discussion.
